docs/product-getting-started-guide/src/main/asciidoc/test-violation-based-proc.adoc

```diff
@@ -0,0 +1,31 @@
+[id='testing_violation_numbers-proc']
+= Testing the multiple violations scenario
+
+The following chapter instructs you how to copy the *Suspend due to total points* asset and modify it to create the driver license suspension scenario for drivers with multiple violations using the data that you specified when you set the Driver License Suspension rules and actions.
```

Comment:
This intro statement is actually pretty good. Just need to nix the "The following chapter instructs" stuff. Depending on how you recast, might need to be two sentences, but maybe not.

Reply:
done